frontend/src/pages/MedDog.tsx

```diff
@@ -1,31 +1,82 @@
 import './MedDog.scss';
-import {LocalizationProvider, StaticDatePicker} from "@mui/lab";
-import AdapterDateFns from '@mui/lab/AdapterDateFns';
-import {TextField} from "@mui/material";
-import React from "react";
+import React, {ChangeEvent, useContext, useState} from "react";
+import AppointmentGallery from "../components/AppointmentGallery";
+import AddCircleOutlineIcon from '@mui/icons-material/AddCircleOutline';
+import {Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField} from "@mui/material";
+import {DateTimePicker, LocalizationProvider} from "@mui/lab";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import {addAppointment} from "../service/RequestService";
+import {AuthContext} from "../context/AuthProvider";
 
 export default function MedDog() {
 
-    const [value, setValue] = React.useState<Date | null>(new Date());
+    const [open, setOpen] = React.useState(false);
+    const [appointment, setAppointment] = useState<string>("")
+    const [dateAndTime, setDateAndTime] = React.useState<Date | null>(new Date());
```

Comment:
```suggestion
    const [open, setOpen] = useState(false);
    const [appointment, setAppointment] = useState<string>("")
    const [dateAndTime, setDateAndTime] = useState<Date | null>(new Date());
```
be consistent

Reply:
You're right, these are some Copy&Paste consts' from mui for the dialog.